### PR TITLE
Add CI pipeline

### PR DESCRIPTION
**Pull request by Kazauwa**
Run tests, validators and linters inside TravisCI pipeline

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  include:
+    - stage: lint
+      env:
+        - HADOLINT: "${HOME}/hadolint"
+      install:
+        - gem install mdl
+        - curl -sL -o ${HADOLINT} "https://github.com/hadolint/hadolint/releases/download/v1.17.6/hadolint-$(uname -s)-$(uname -m)"
+        - chmod 700 ${HADOLINT}
+      script:
+        - mdl README.md
```
Comment (Melevir) [inline .travis.yml:20]: `make chech` seens to be missing

Comment (Kazauwa) [reply .travis.yml:20]: It is in a different job down below. This job only validates README.md and Dockerfile.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        - postgresql
+        - redis
+      install:
+        - pip install -r requirements.txt
+        - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+        - chmod +x ./cc-test-reporter
+      before_script:
+        - ./cc-test-reporter before-build
+      script:
+        - make check
```
Comment (Kazauwa) [inline .travis.yml:42]: `make check` is here

File: .travis.yml (hunk 1 above)
Comment (Melevir) [reply .travis.yml:20]: Seems weird: `make check` does linting, not testing.

But ok.

**Pull request merged**